### ISSUE TITLE
ceod TURN server request uses ancient format

### ISSUE THREAD

**Issue by fippo** [webrtc/apprtc#245]
https://computeengineondemand.appspot.com/turn?username=322879358&key=4080218913 still returns the turn server urls as "uris":
```
{"username": "1450520268:322879358", "password": "...", "uris": ["turn:104.155.26.71:3478?transport=udp", "turn:104.155.26.71:3478?transport=tcp", "turn:104.155.26.71:3479?transport=udp", "turn:104.155.26.71:3479?transport=tcp"]}
```
Also uses 'password' instead of 'credentials which makes it necessary to translate the format before passing it to the peerconnection here https://github.com/webrtc/apprtc/blob/master/src/web_app/js/util.js#L96-L100

cc @KaptenJansson

**Comment by KaptenJansson** [comment]
I assume this is because the turn server follows http://tools.ietf.org/html/draft-uberti-behave-turn-rest-00#section-2.2.

@juberti what's your take on this?

**Comment by KaptenJansson** [comment]
This will be fixed in #219
